### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by SpecKeef**
## Just little improvements

- Added some consistent variables

## Description

 Added some consistent variables for consistency

# How Has This Been Tested?

No visual changes.

### CODE REVIEW

**Comment by RacoonDog** [comment]
`MeteorCleint.NAME` already exists

**Comment by SpecKeef** [comment]
> `MeteorCleint.NAME` already exists

yeah lol i just realized.

**Comment by Big-Iron-Cheems** [comment]
I have no idea why you retouch so much code unnecessarily...

- You added `else` to `if` that already end with a return for no reason.
- You then mangled the settings syntax in a way that is inconsistent from the codebase.
- You also proceed to put asserts that are unnecessary, and add parentheses to single line statements.
- And finally you inverted guard statements, reducing readability and adding more indents.

Overall, I have no idea what the point of all this is, and how is it related with the original task of this PR.

File: src/main/java/meteordevelopment/meteorclient/systems/modules/render/EntityOwner.java [hunk 1]
```diff
@@ -38,26 +38,30 @@ public class EntityOwner extends Module {
...
+            .min(0)
+            .build()
+        );
+
+        projectiles = sgGeneral.add(new BoolSetting.Builder()
+            .name("projectiles")
+            .description("Display owner names of projectiles.")
+            .defaultValue(false)
+            .build()
+        );
```
Comment (Fiz-Victor) [inline src/main/java/meteordevelopment/meteorclient/systems/modules/render/EntityOwner.java:64]: why init settings in instantiation

**Review by Fiz-Victor — CHANGES_REQUESTED** [review]

**Pull request closed without merging**